app/src/components/testCaseList/testCaseCard/testCaseCard.jsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { useIntl } from 'react-intl';
+import Parser from 'html-react-parser';
+import { PopoverControl } from 'pages/common/popoverControl';
+import { TagList } from 'pages/inside/productVersionPage/linkedTestCasesTab/tagList';
+import PriorityIcon from 'common/img/newIcons/priority-inline.svg';
+import { messages } from './messages';
+import styles from './testCaseCard.scss';
+
+const cx = classNames.bind(styles);
+
+const StatusIcon = ({ status }) => {
+  const iconMap = {
+    passed: <div className={cx('priority-icon')}>{Parser(PriorityIcon)}</div>,
+    failed: '✗',
+    skipped: '○',
+    in_progress: '⟳',
```

Comment:
let's add the icons to ui-kit in svg format and import from there

Reply:
fixed